src/web/index.php

```diff
@@ -10,19 +10,42 @@
  * (see Filtering tasks 1 and 2 below)
  */
 
+$get = $_GET;
+
+if (isset($_GET['filter_by_first_letter'])) {
+    $airports = filterByFirstLetter($airports, $get['filter_by_first_letter']);
+}
+
+if (isset($_GET['filter_by_state'])) {
+    $airports = filterByState($airports, $get['filter_by_state']);
+}
+
 // Sorting
 /**
  * Here you need to check $_GET request if it has sorting key
  * and apply sorting
  * (see Sorting task below)
  */
 
+if (isset($_GET['sort'])) {
+    $airports = sortAirports($airports, $get['sort']);
+}
+
 // Pagination
 /**
  * Here you need to check $_GET request if it has pagination key
  * and apply pagination logic
  * (see Pagination task below)
  */
+
+isset($_GET['page']) ? $currentPage = intval($_GET['page']) :   $currentPage = 1;
```

Comment:
Please change it to
`$currentPage = isset($_GET['page']) ?  intval($_GET['page']) :  1;`